smtpd.go

```diff
@@ -713,7 +713,7 @@ func (s *session) readData() ([]byte, error) {
 func (s *session) makeHeaders(to []string) []byte {
 	var buffer bytes.Buffer
 	now := time.Now().Format("Mon, _2 Jan 2006 15:04:05 -0700 (MST)")
-	buffer.WriteString(fmt.Sprintf("Received: from %s (%s [%s])\r\n", s.remoteName, s.remoteHost, s.remoteIP))
+	buffer.WriteString(fmt.Sprintf("Received: from %s\r\n", s.remoteName))
```

Comment:
Sorry, the IP address removal was an unrelated change that I pushed to this PR by accident. I figured the original PR was not getting merged, so I closed it to avoid confusion. Sorry for confusion :-/